src/index.js

```diff
@@ -27,8 +27,19 @@ function PaymentsClient(config) {
     throw new Error('paymentHost must be http or https');
   }
 
-  if (typeof this.product !== 'string') {
-    throw new Error('A \'product\' string must be provided');
+  if (typeof config.product !== 'object' || !config.product.id) {
+    throw new Error('A product id must be provided in a product object');
+  }
+
+  var productImageUrl = config.product.image;
+  if (!productImageUrl) {
+    console.warn(
+      'A product image URL was not supplied as a property of config.product.');
```

Comment:
Can we fall back to the Firefox logo if no image is supplied? It will look wierd without an image.